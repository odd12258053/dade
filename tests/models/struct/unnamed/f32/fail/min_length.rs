use dade::model;
#[model]
struct TestModel (
   #[field(min_length = 2)]
    f32
);
fn main() {}
