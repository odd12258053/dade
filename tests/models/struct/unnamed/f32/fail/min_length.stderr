error: Support condition is gt, ge, lt, le, alias, default and validate
 --> tests/models/struct/unnamed/f32/fail/min_length.rs:2:1
  |
2 | #[model]
  | ^^^^^^^^
  |
  = note: this error originates in the attribute macro `model` (in Nightly builds, run with -Z macro-backtrace for more info)
